hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -246,6 +253,249 @@ def queries(query_group,
     return ret
 
 
+@hubble_status.watch
+def osqueryd_monitor(configfile=None,
+                     flagfile=None,
+                     logdir=None,
+                     databasepath=None,
+                     pidfile=None,
+                     hashfile=None,
+                     daemonize=True):
+    '''
+    This function will monitor whether osqueryd is running on the system or not. 
+    Whenever it detects that osqueryd is not running, it will start the osqueryd. 
+    Also, it checks for conditions that would require osqueryd to restart(such as changes in flag file content) 
+    On such conditions, osqueryd will get restarted, thereby loading new files.
+
+    configfile
+        Path to osquery configuration file.
+
+    flagfile
+        Path to osquery flag file
+
+    logdir
+        Path to log directory where osquery daemon/service will write logs
+
+    pidfile
+        pidfile path where osquery daemon will write pid info
+
+    hashfile
+        path to hashfile where osquery flagfile's hash would be stored
+
+    daemonize
+        daemonize osquery daemon. Default is True. Applicable for posix system only
+
+    '''
+    log.info("starting osqueryd monitor")
+    saltenv = __salt__['config.get']('hubblestack:nova:saltenv', 'base')
+    osqueryd_path = 'salt://osqueryd'
+    cached = __salt__['cp.cache_dir'](osqueryd_path,saltenv=saltenv)
+    log.info('cached osqueryd files to cachedir')
+    cachedir = os.path.join(__opts__.get('cachedir'),'files',saltenv,'osqueryd')
+    base_path = cachedir
+    servicename = "hubble_osqueryd"
+    if not logdir:
+            logdir = __opts__.get('osquerylogpath')
+    if not databasepath:
+            databasepath = __opts__.get('osquery_dbpath')
+    if salt.utils.platform.is_windows():
+        log.info("System is windows")
```

Comment:
We can probably get rid of these logs (this line and on the `else` as well). In general I don't think we're wondering whether a system is windows or linux when we're looking at logs, and `.info` should be reserved for more significant info.

Reply:
sure, I'll remove these logs